apps/examples/testcase/le_tc/drivers/tc_adc.c

```diff
@@ -0,0 +1,116 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_adc.c
+/// @brief Test Case Example for adc driver
+#include <tinyara/analog/adc.h>
+#include <tinyara/analog/ioctl.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include "tc_internal.h"
+#include <errno.h>
+
+/**
+ * @brief Closes all open files
+ */
+static void close_fds(int *fds, int count)
+{
+	for (; count >= 0; count--) {
+		close(fds[count]);
+	}
+}
+
+/**
+* @fn                   :tc_adc_open_close
+* @brief                :Test the adc driver open and close
+* @scenario             :Test the adc driver open and close
+* API's covered         :open, close
+* Preconditions         :none
+* Postconditions        :none
+* @return               :void
+*/
+static void tc_adc_open_close(void)
```

Comment:
Because our naming rule of tc is ```<TC_Category>_<Module>_<FunctionName>_<Positive/Negative>_<AdditionalInformation>```,  please add "driver" like below.
```
tc_driver_adc_open_close
```

```<Positive/Negative>_<AdditionalInformation>``` is not mandatory on le_tc so that you don't need to concern.

Reply:
Done